thirdparty/CMakeLists.txt

```diff
@@ -35,14 +35,15 @@ if(NOT spdlog_POPULATED)
 endif()
 
 ###################################################################################################
-# - thrust ----------------------------------------------------------------------------------------
+# - thrust/cub ------------------------------------------------------------------------------------
```

Comment:
I'd just leave it at thrust, because cub isn't a direct dependencies, and whatever thrust does internally, we don't really need to care aobut.

Reply:
See above.